addon-test-support/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as start } from './start';
+export { default as startEmberExam } from './start';
```

Comment:
Why are you duplicating the name here? 🤔 I would just pick one, probably `start`

Reply:
I think it's fine having ember exam reflect what we're doing in ember-qunit 👍 I think potentially better 🎉 

Could you remove this and change the other thing to `{start as startEmberExam}` ?

Reply:
updated